org.gnome.Fractal.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id": "org.gnome.Fractal",
+    "version": "master",
```

Comment:
Please remove it as it has no effect on the resulting flatpak; it's better to put version information into appdata.